posts/2024-11-10-investigating-spirv-for-the-shader-system-part-2.md

```diff
@@ -0,0 +1,252 @@
+---
+title: "Investigating SPIR-V for the shader system - Part 2"
+author: youness
+popular: false
+image: /images/spir.png
+tags: ['.NET', 'Shaders']
+---
+This second part is going to dive deeper in how the current SDSL compiler works and how we are improving on it for the SPIR-V compiler. This will be a sort of personal log book of my research on the subject.
+
+---
+
+Table of Contents:
+
+[[TOC]]
+
+In the last blog post we've briefly went through the why SPIR-V was an interesting shader format for our new shader compiler. After a year of research on this subject we're coming back with ideas and implementations !
```

Comment:
I know it is still WIP but I have some comments 🙈.

This could be a link to the post Part 1..
> In the last [blog post](https://www.stride3d.net/blog/investigating-spirv-for-the-shader-system/)

Also, you can edit the post Part 1, and link to Part 2, so the reader is aware of the follow up article.

Interlinking existing posts, content in Manual or any existing content is very helpful for the readers.

Reply:
Sure, will do that!